docroot/themes/humsci/humsci_basic/src/scss/components/_views-exposed-form.scss

```diff
@@ -1,31 +1,51 @@
-.hb-layout-builder-main-content .views-exposed-form {
+.hb-three-column__main .views-exposed-form,
+.hb-three-column__sidebar-1 .views-exposed-form {
   display: flex;
-  align-items: flex-end;
+  align-items: flex-start;
```

Comment:
It looks like this is impacting the exposed form on the news page...
![image](https://user-images.githubusercontent.com/1256329/81735617-4f388a00-9463-11ea-9187-624d91f5227c.png)

Reply:
fixed!